runconfig/merge.go

```diff
@@ -20,6 +20,17 @@ func Merge(userConf, imageConf *Config) error {
 	if userConf.CpuShares == 0 {
 		userConf.CpuShares = imageConf.CpuShares
 	}
+
+	// FIXME: Config.Cpuset is obsoleted, will be removed.
+	if userConf.CpusetCpus != "" {
+		userConf.Cpuset = userConf.CpusetCpus
+	} else if userConf.Cpuset != "" {
+		userConf.CpusetCpus = userConf.Cpuset
+	} else {
+		userConf.Cpuset = imageConf.Cpuset
+		userConf.CpusetCpus = imageConf.CpusetCpus
+	}
+
```

Comment:
Can you explain more why we don't want these fields to appear in the image?
From what I understanding, this part is for `POST /commit`, users can send a json
file in the request which might contain the old cpuset entry, that was supposed to
be added in the image, right? 
And users can inspect an image through `GET /images/(name)/json`, they might
want to get their old `Cpuset` entry.